src/scss/pages/_phonebook.scss

```diff
@@ -1,3 +1,43 @@
+.phonebook-results {
+  font-size: .9rem;
+
+  .toggle {
+    &:not(.collapsed) {
+      .fa-plus {
+        display: none;
+      }
+    }
+
+    &.collapsed {
+      .fa-minus {
+        display: none;
+      }
+    }
+  }
+
+  .result-label {
+    font-size: 85%;
+  }
+
+  .email,
+  .phone {
+    text-overflow: ellipsis;
```

Comment:
For `text-overflow` to work as expected, you'll need to force the text in these elements to overflow from their containers by adding:
```
overflow: hidden;
white-space: nowrap;
```

see https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Reply:
those styles were added along with `display: block;` to make that work correctly. Thanks!